Automattic/wp-calypso
# Nextdoor: Add the connection to Calypso

## Proposed Changes

* Add placeholder image
* Add Nextdoor example UI
* Remove "NEW" badge from Mastodon and Instagram

## Testing Instructions



* Apply D125526-code to your sandbox
* Point `public-api.wordpress.com` to your sandbox
* Add the blog sticker to your site:
`add_blog_sticker( 'jetpack-social-nextdoor-connection', null, null, <YOUR_SITE_ID> );` 
* Goto `/marketing/connections/:site` in Calypso Blue
* Confirm that you see the Nextdoor connection
* Expand the Nextdoor connection along with the description _"Share posts to Nextdoor."_
* Confirm that you see the placeholder image and the text _"Automatically share posts to Nextdoor."_
* Confirm that you see the "NEW" badge for Nextdoor
* Confirm that you no longer see the "NEW" badge for Mastodon and Instagram
* Goto `/jetpack-social/:site` on Calypso Green and confirm the above steps


Calypso Blue

<img width="1070" alt="Screenshot 2023-10-19 at 11 27 07 AM" src="https://github.com/Automattic/wp-calypso/assets/18226415/92380d50-ae30-4f31-a556-69eaf271e794">
<img width="1067" alt="Screenshot 2023-10-19 at 11 27 23 AM" src="https://github.com/Automattic/wp-calypso/assets/18226415/6e882e06-0de0-4d81-8701-f02331446b41">

Calypso Green


<img width="1047" alt="Screenshot 2023-10-19 at 11 27 47 AM" src="https://github.com/Automattic/wp-calypso/assets/18226415/7217d817-3312-40d1-acbe-6fdc344614f8">
<img width="1118" alt="Screenshot 2023-10-19 at 11 27 58 AM" src="https://github.com/Automattic/wp-calypso/assets/18226415/79dbbf44-5c27-494e-b6aa-c6f1b2c13ec9">


## Pre-merge Checklist
